enterprise/api/v1/serializers.py

```diff
@@ -1735,7 +1735,7 @@ class EnterpriseGroupLearnersRequestQuerySerializer(serializers.Serializer):
     show_removed = serializers.BooleanField(required=False, default=False)
     is_reversed = serializers.BooleanField(required=False, default=False)
     page = serializers.IntegerField(required=False)
-    lms_users = serializers.ListField(
-        child=serializers.IntegerField(required=True),
+    learners = serializers.ListField(
+        child=serializers.EmailField(required=True),
```

Comment:
[suggestion] can we update the `get_learners` docstring in `enterprise/api/v1/views/enterprise_group.py` to include this required query param?